js/researches/german/GermanParticiple.js

```diff
@@ -0,0 +1,83 @@
+var Participle = require( "../../values/Participle.js" );
+
+var getIndices = require( "../../stringProcessing/indices.js" ).getIndices;
+var getIndicesOfList = require( "../../stringProcessing/indices.js" ).getIndicesOfList;
+var exceptionsRegex =
+	/\S+(apparat|arbeit|dienst|haft|halt|keit|kraft|not|pflicht|schaft|schrift|tät|wert|zeit)($|[ \n\r\t\.,'\(\)\"\+\-;!?:\/»«‹›<>])/ig;
+var exceptionsParticiplesActive = require( "./passivevoice-german/exceptionsParticiplesActive.js" )();
+
+var forEach = require( "lodash/forEach" );
+var includes = require( "lodash/includes" );
+var map = require( "lodash/map" );
+
+/**
+ * Creates an Participle object for the English language.
+ *
+ * @param {string} participle The participle.
+ * @param {string} sentencePart The sentence part that contains the participle.
+ * @param {object} attributes  The attributes object.
+ *
+ * @constructor
+ */
+var GermanParticiple = function(  participle, sentencePart, attributes ) {
+	Participle.call( this, participle, sentencePart, attributes );
+	this.checkException();
+};
+
+require( "util" ).inherits( GermanParticiple, Participle );
+
+/**
+ * Sets sentence part passiveness to passive if there is no exception.
+ *
+ * @returns {void}
+ */
+GermanParticiple.prototype.checkException = function() {
```

Comment:
`checkException` feels weird, because this method is setting a value. If you put the line `isPassive` in its own method you can use `this.setSentencePartPassiveness( isPassive );` directly at the places you are calling `checkException`